example-workflows/README.md

```diff
@@ -1,2 +1,15 @@
 # Example Workflows
 
+Here are some tutorial notebooks showcasing the various features of Aqueduct:
+
+* [Quickstart](example-workflows/quickstart-tutorial.md)
```

Comment:
As I mentioned on the other PR, I'd prefer to just keep one quickstart rather than have two.

Reply:
Updated the quickstart guide and translated it into a notebook format here.